cscore/src/main/native/include/cscore_cpp.h

```diff
@@ -21,6 +21,12 @@
 
 #include "cscore_c.h"
 
+#ifdef _WIN32
```

Comment:
Shouldn’t these ifdefs be checking for MSVC (_MSC_VER) rather than _WIN32?  I guess we don’t support any other compilers on Windows, but...

Reply:
Clang on windows runs the same flags, and we use _WIN32 everywhere else, so if we change we should change everything.